### PR TITLE
Feature/ws-notifiaction

### PR DESCRIPTION
**Pull request by NikPaukov**

### CODE REVIEW

**Review by DavidKorystov — APPROVED** [review]

File: src/config/ws.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    return;
+  }
+  const userId: string = req.headers.userid as string;
+  const arrayOfConnections: Set<WebSocket> = connectedUsers.get(userId);
+  if (arrayOfConnections) {
+    arrayOfConnections.add(ws);
+    connections.set(ws, userId);
+  } else {
+    try {
+      const user = (await getList({ _id: userId }, {})).docs[0];
```
Comment (VitaliiDenysiuk) [inline src/config/ws.ts]: use getUserById

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]
When I try connect frontend to your branch with clean data: frontend does not work, with other branch all ok.

File: src/config/ws.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+import { getList } from "../repositories/user.repository";
+import { IncomingMessage } from "http";
+
+export const connectedUsers: Map<string, Set<WebSocket>> = new Map();
+const connections: Map<WebSocket, string> = new Map();
+
+export const connection = async (ws: WebSocket, req: IncomingMessage) => {
+  if (!req.headers.userid) {
```
Comment (VitaliiDenysiuk) [inline src/config/ws.ts]: why do we add it at header? Maybe better use 
ws.on("message"  and put it in that section. userid can put inside message

File: src/config/ws.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    ws.close(401, "Id Required");
+    return;
+  }
+  const userId: string = req.headers.userid as string;
```
Comment (VitaliiDenysiuk) [inline src/config/ws.ts]:
```suggestion
  const userId: string = `${req.headers.userid}`;
```

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]

**Pull request merged**